doc/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -59,7 +59,7 @@ To register a listener in a laminas-mvc based application, use either applicatio
 
 This example uses module configuration, e.g. `module/Application/config/module.config.php`:
 
-<pre class="language-php" data-line="4-6"><code>
+```php
```

Comment:
@gsteel 
You have removed the line highlighting: https://docs.laminas.dev/laminas-eventmanager/application-integration/usage-in-a-laminas-mvc-application/#register-listener